HISTORY.md

```diff
@@ -6,6 +6,10 @@
 ### Behavior Changes
 * Best-efforts recovery ignores CURRENT file completely. If CURRENT file is missing during recovery, best-efforts recovery still proceeds with MANIFEST file(s).
 * In best-efforts recovery, an error that is not Corruption or IOError::kNotFound or IOError::kPathNotFound will be overwritten silently. Fix this by checking all non-ok cases and return early.
+* If `share_table_files` and `share_files_with_checksum` are true, BackupEngine will compare the checksums computed before and after copying of the table files.
+* When `file_checksum_gen_factory` is set to `GetFileChecksumGenCrc32cFactory()`, BackupEngine will calculate the crc32c checksum of the table file to be backed up and compare the resulting value with the one stored in the DB manifest before copying the file to the backup directory. In addition, after copying the table file to the backup directory, BackupEngine will double-check the checksum of the copied file with the one calculated before copying. If the `file_checksum_gen_factory` is not set or set to any other customized factory, checksum verification with DB manifest before copying will not be performed.
```

Comment:
Related to above, I suggest this:

* When `file_checksum_gen_factory` is set to `GetFileChecksumGenCrc32cFactory()`, BackupEngine will compare the crc32c checksums of table files computed when creating a backup to the expected checksums stored in the DB manifest, and will fail CreateNewBackup on mismatch (corruption). If the `file_checksum_gen_factory` is not set or set to any other customized factory, there is no checksum verification to detect if SST files in a DB are corrupt when read, copied, and independently checksummed by BackupEngine.